lib/github_service/commands/add_label.rb

```diff
@@ -3,9 +3,9 @@ module Commands
     class AddLabel < Base
       include IsTeamMember
 
-      UNASSIGNABLE = {
-        "jansa/yes" => "jansa/yes?"
-      }.freeze
+      def unassignable_labels
```

Comment:
Same here

Reply:
It's already cached in `Settings` and that should get reloaded any time the settings are changed.  Are you concerned about the call to `stringify_keys`?

Reply:
By that logic, then we just shouldn't be caching it at all, and makes me wonder why you have a `@` var there in the first place.

Reply:
I added the cache here so that we don't have to call `stringify_keys` for every label in the command.

Reply:
The other cache was not necessary, so I removed it.

Reply:
Sorry, to be clear:

I don't really see this as a big performance hit, since we would be caching it for it's once use per instance of this (it seems), so I am more commenting on the fact that we are caching this on the instance, but it's cache should never be made use of since this is only ever called once per instance.

If we still want the cache, then probably `cache_with_timeout` would make sense based on the use case that you described:  changing the files and having it auto reload without a server restart.  That said, I have found that the `listen` gem has caused my CPU to spike to 100% on my current machine, so I tend to avoid trying to rely on that functionality.

Reply:
> I added the cache here so that we don't have to call stringify_keys for every label in the command.

Correction:  You are right with this since it is being called with in a loop.  I would argue that it is negligible with the number of labels we currently have in this list and the number of labels that are passed in for `add_label` in general (one or two), and there is more savings to be had by caching this on a class level is that is your true intent.

Reply:
> That said, I have found that the listen gem has caused my CPU to spike to 100% on my current machine, so I tend to avoid trying to relying on that gem/functionality.

Can you open an issue for this...I'd not heard this before and I don't have this issue nor do we see it in prod.

Reply:
I think the memoization is fine since this is at an instance level.  The next instance of AddLabel will query the Settings again.